client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -175,19 +196,25 @@ export default React.createClass( {
 			return;
 		}
 
+		const eventName = 'google_analytics_' + abtest( 'contextualGoogleAnalyticsNudge' ) || 'google_analytics_notice';
+
 		debug( 'Google analitics is not enabled. adding nudge ...' );
 
 		return (
 			<UpgradeNudge
 				title={ this.translate( 'Add Google Analytics' ) }
 				message={ this.translate( 'Upgrade to the business plan and include your own analytics tracking ID.' ) }
 				feature="google-analytics"
-				event="google_analytics_notice"
+				event={ eventName }
```

Comment:
`settingsDisabledFeature` should go to `/plans/features/google-analytics/artpitesting.wordpress.com`
`settingsDisabledPlansCompare` should go to `/plans/artpitesting.wordpress.com?feature=google-analytics`